notebooks/data-sources/Bugzilla/bugzilla_EDA.ipynb

```diff
@@ -0,0 +1,3552 @@
+{
```

Comment:
<p>setp -&gt; step? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/243/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Oops yeah that's a typo :)</p>